source/index.md

```diff
@@ -713,13 +713,53 @@ No parameters are needed to make a new void transaction.
 ### Credits
 
 To payout (e.g. winnings and not refunds) money to a cardholder's bank account
-you make a new credit resource. You must have a card resource to make a credit
-transaction.
+you make a new credit resource.
 
 #### Parameters
 
 ````
-POST https://gateway.clearhaus.com/cards/:id/credits
+POST https://gateway.clearhaus.com/credits
+````
+
+<dl class="dl-horizontal">
+  <dt>amount</dt>
+  <dd>[1-9][0-9]{0,9} <br />
```

Comment:
We use `[0-9]{1,10}` for amount for `/authorizations` now :slightly_smiling_face:

Reply:
Of course, we will have to adapt https://github.com/clearhaus/klovedal/blob/5f097d212b9f5d87724ab5ceb0c67f2b13e8f7b4/lib/klovedal/request/credit.rb#L8

Reply:
"0-credits" should not be allowed; 0-auths are allowed.

Reply:
Do'h. Good point.